frontend/src/app/features/team-planner/team-planner/planner/team-planner.component.html

```diff
@@ -25,6 +25,19 @@
       data-qa-selector="add-existing-pane"
     >
     </op-add-existing-pane>
+
+    <button
+      class="button button_no-margin op-team-planner--view-select-dropdown"
```

Comment:
Better to remove the `no-margin` class and set `margin: 0;` in the `op-team-planner--view-select-dropdown` rules

Reply:
Done